### ISSUE TITLE
AppPixivAPI.user_illusts() cannot ignore "type" parameter.

### ISSUE THREAD

**Issue by Xdynix** [upbit/pixivpy#38]
Try
```python
    def user_illusts(self, user_id, illust_type=None, illust_filter='for_ios', offset=None, req_auth=False):
        url = 'https://app-api.pixiv.net/v1/user/illusts'
        params = {
            'user_id': user_id,
            'filter': illust_filter
        }
        if illust_type:
            params['type'] = illust_type
        if offset:
            params['offset'] = offset
        r = self.no_auth_requests_call('GET', url, params=params, req_auth=req_auth)
        return self.parse_result(r)
```

If `type` not in `params`, every type of illust can be fetched, including ugoria.

**Comment by upbit** [comment]
I will add this parameter as soon as possible.
BTW: Do you have `type` definition for `user/illusts`?

**Comment by Xdynix** [comment]
Unfortunately, no. I found this feature by trying it casually.
The `filter` parameter can also be ignored or use `for_android`. But it doesn't seem to make sense.

**Comment by steinuil** [comment]
As far as I know, `type` can take `illust`, `manga`, `novel` or `ugoira`.

**Comment by Xdynix** [comment]
@steinuil 
`novel` and `ugoira` is invalid.

Response will be:
```python
{'error': {'message': '{"type":["Type contains invalid value"]}',
           'reason': '',
           'user_message': '',
           'user_message_details': {}}}
```

**Comment by steinuil** [comment]
Oh, you're right, I got confused. There's a separate endpoint for novels, it's `v1/user/novels` I think.

**Comment by Xdynix** [comment]
I just found that my code above is not good.

I've changed parameter `type` and `filter` into `illust_type` and `illust_filter` to avoid IDE inspection warning("Shadows built-in name").

But it will make the result of `AppPixivAPI.parse_qs()` unable to be used as kwargs.

**Comment by upbit** [comment]
Fixed in [6d6bf51](https://github.com/upbit/pixivpy/commit/6d6bf51ec859e524c724989b92e7aaab08bfa228)

Keep consistent with the old API, type param default is `illust`. Try it:

```py
    aapi = AppPixivAPI()
    for illust in aapi.user_illusts(2319374, type=None).illusts:
        print illust.id, illust.type
```

```
62406621 manga
62303343 manga
62206856 illust
61845062 manga
61765261 manga
61201232 illust
60306534 manga
59361415 manga
59135433 manga
59134642 manga
57954791 illust
57709493 illust
57350442 illust
57349586 illust
57250114 manga
56622927 manga
55985523 manga
55955863 illust
53666521 illust
22455282 illust
21051841 illust
17367177 illust
```

**Comment by upbit** [comment]
This commit included in [v3.3.3](https://github.com/upbit/pixivpy/tree/e2008440440336b59b952e5136e7dd49d9665b43)
